crates/storage/Cargo.toml

```diff
@@ -10,7 +10,7 @@ rust-version = { workspace = true }
 [dependencies]
 anyhow = { workspace = true }
 base64 = { workspace = true }
-bincode = "2.0.0-rc.3"
+bincode = { version = "2.0.0-rc.3", features = ["serde"] }
```

Comment:
Something to also consider is that some felt values are often real numbers like `0x1` etc which means encoding the `Felt` could benefit from skipping leading zeros instead of encoding `[u8;32]` entirely.

Reply:
@kkovaacs do you think its worth compressing/zero-skipping felts, or will zstd just handle that for us somewhat? i.e. `&[u8]` vs `[u8; 32]`?

Reply:
I'd bet zstd does take care of that so I wouldn't expect big differences.

Reply:
That's even better!